clients/ui/frontend/src/app/pages/modelRegistry/screens/ModelPropertiesDescriptionListGroup.tsx

```diff
@@ -14,18 +14,16 @@ const ModelPropertiesDescriptionListGroup: React.FC<ModelPropertiesDescriptionLi
   isArchive,
   saveEditedCustomProperties,
 }) => (
-  <DashboardDescriptionListGroup
-    title=""
-    isEmpty={Object.keys(customProperties).length === 0}
-    contentWhenEmpty="No properties"
-  >
-    <ModelPropertiesExpandableSection
-      customProperties={customProperties}
-      isArchive={isArchive}
-      saveEditedCustomProperties={saveEditedCustomProperties}
-      isExpandedByDefault
-    />
-  </DashboardDescriptionListGroup>
+  <DescriptionListGroup>
```

Comment:
Do we even need `DescriptionListGroup` and `DescriptionListDescription`?

Reply:
@DaoDaoNoCode I think we should maybe look into that as a followup task so we don't block @Taj010's PR? Maybe we don't close the issue just yet if that's necessary

Reply:
@mturley A follow-up task sounds good to me!